app/mailers/mailer.rb

```diff
@@ -7,8 +7,8 @@ class Mailer < ApplicationMailer
   default_url_options[:host] = Gemcutter::HOST
   default_url_options[:protocol] = Gemcutter::PROTOCOL
 
-  def email_reset(user)
```

Comment:
why not still accept the user?

Reply:
Yup, I'll update this back.

Reply:
Speaking of, what happens if the record disappears after it's enqueued but before it's processed? Is that a DeserializationError?

Reply:
yes, DeserializationError